Source/Core/VideoBackends/Null/NullTexture.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "VideoCommon/AbstractTexture.h"
+#include "VideoCommon/VideoCommon.h"
+
+namespace Null
+{
+
+class NullTexture final : public AbstractTexture
+{
+public:
+  explicit NullTexture(const TextureConfig& config);
+  ~NullTexture();
+
+  virtual void Bind(unsigned int stage) override;
```

Comment:
Yes. The rule we use is that we don't write `virtual` if there also is an `override`.